### DISCUSSION TITLE
Share buttons are not showing

### DISCUSSION THREAD

**Discussion by sandsoncosta** [hbstack/.github#90] [Issues and bugs]
![image](https://github.com/user-attachments/assets/8af997bd-4739-4ef7-8793-9e5850b2e3e8)
![image](https://github.com/user-attachments/assets/0423c40e-195f-4264-8017-fa57525c3def)

**Comment by razonyang** [comment]
You may using a very old version of that module, please update modules and try again.

```sh
hugo mod get -u github.com/hbstack/social-share-buttons
```
